config.ini

```diff
@@ -29,15 +29,19 @@
 enable_database = yes
 drop_database = no
 ;
+;  PORT FROM CRIB SHEET
```

Comment:
What does port from crib sheet mean?

Reply:
I'm following the port mappings from marks developer environment, it shouldn't be there as a comment. I'll remove it